### ISSUE TITLE
Helper method with void return type causes Fatal Error

### ISSUE THREAD

**Issue by emulgeator** [Codeception/Codeception#5885]
#### What are you trying to achieve?
I have helper methods what are declared with a void  return type. For example:
```php
public function doSomething(): void
{
    // Do something without a return statement
}
```

The build should generate the proxy method in the actor class something like this:
```php
public function clearDbTables($tables = null): void {
    $this->getScenario()->runStep(new \Codeception\Step\Action('doSomething', func_get_args()));
}
```

#### What do you get instead?
It generates this:
```php
public function clearDbTables($tables = null): void {
    return $this->getScenario()->runStep(new \Codeception\Step\Action('doSomething', func_get_args()));
}
```
Which results:
`Fatal error: A void function must not return a value in test/support/_generated/TesterActions.php on line X`

### Details

* Codeception version: 4.1.3
* PHP Version: 7.2.15
* Operating System: Linux
* Installation type: Composer

**Comment by Naktibalda** [comment]
Fixed by #5880

**Comment by Naktibalda** [comment]
Fixed in 4.1.4
